client/src/js/components/bhBreadcrumb.js

```diff
@@ -95,9 +95,12 @@ function BreadcrumbController() {
    */
   vm.bcDropdowns = vm.dropdown || [];
 
-  /** call the appropriate function and update the dropdown label */
+  vm.dropDownItemSelectedLabel = ''; //for the test purpose
```

Comment:
Hmm.  It usually isn't good practice to add variables into code simply for test purposes, unless there is an easy way to remove them in a `production` build.  Is there any way else to test this component without adding this tag?   Maybe use `child.action()`'s call to trigger an event and detect the event in the test?